win32/src/PyUnicode.cpp

```diff
@@ -101,7 +101,6 @@ BOOL PyWinObject_AsChars(PyObject *stringObject, char **pResult, BOOL bNoneOK /*
     }
     // Convert the string if a WIDE string.
     if (PyUnicode_Check(stringObject)) {
-        // PyUnicode_EncodeMBCS was removed in Py 3.11.
         stringObject = tempObject = PyUnicode_AsMBCSString(stringObject);
```

Comment:
Reducing changed files
```suggestion
        // PyUnicode_EncodeMBCS was removed in Py 3.11.
        stringObject = tempObject = PyUnicode_AsMBCSString(stringObject);
```